frontend_react/src/components/StatsSummary/StatsSummary.tsx

```diff
@@ -197,20 +315,58 @@ const StatsSummary = () => {
                   </MenuItem>
                 ))}
               </Select>
+              <FormGroup style={{marginLeft: "auto"}}>
+                <FormControlLabel labelPlacement="top" control={
+                  <Switch onClick={handleChartVisibility}/>
+                } label="Chart" />
```

Comment:
Can we use an InputLabel for 'Charts' here as well to match the format of line 303?